manifest.json

```diff
@@ -10,9 +10,11 @@
         "48": "images/icon-48.png",
         "128": "images/icon-128.png"
     },
-
+    "background": {
+        "scripts": ["./background.js"]
+    },
     "browser_action":{
-        "default_popup": "popup.html",
+        "default_popup": "login.html",
```

Comment:
I'm surprised that you don't have to list all of the added JavaScript files here. It's been a while since I've written an extension but I vaguely recall that all of the resources in the extension need to be listed. Check this if you're not yourself sure.